assets/mariadbapp/main.go

```diff
@@ -0,0 +1,89 @@
+/*
+   Copyright 2020 SUSE
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/cloudfoundry-community/go-cfenv"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+var serviceName string
+
+func init() {
+	serviceName = os.Getenv("SERVICE_NAME")
```

Comment:
```suggestion
	serviceName = os.Getenv("SERVICE_NAME")
	if serviceName == "" {
		panic("SERVICE_NAME not set")
	}
```
Though I guess we'd fall over in `main()` anyway so maybe it doesn't matter? It would be nice to note _somewhere_ that the issue is because the name was unset though.

It's unclear why we need to do this in `init()` rather than in `main()`.